op-acceptance-tests/tests/jovian/min_base_fee_test.go

```diff
@@ -1,270 +1,150 @@
 package jovian
 
 import (
-	"context"
-	"crypto/ecdsa"
 	"math/big"
 	"testing"
 	"time"
 
-	"github.com/ethereum-optimism/optimism/devnet-sdk/system"
-	"github.com/ethereum-optimism/optimism/devnet-sdk/testing/systest"
-	"github.com/ethereum-optimism/optimism/devnet-sdk/testing/testlib/validators"
-	"github.com/ethereum-optimism/optimism/devnet-sdk/types"
-	"github.com/ethereum-optimism/optimism/op-e2e/bindings"
+	"github.com/ethereum-optimism/optimism/op-chain-ops/devkeys"
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/dsl"
+	"github.com/ethereum-optimism/optimism/op-devstack/presets"
 	"github.com/ethereum-optimism/optimism/op-node/rollup"
-	"github.com/ethereum/go-ethereum/accounts/abi/bind"
-	gethTypes "github.com/ethereum/go-ethereum/core/types"
-	"github.com/ethereum/go-ethereum/ethclient"
-	"github.com/ethereum/go-ethereum/params"
-	"github.com/stretchr/testify/require"
+	"github.com/ethereum-optimism/optimism/op-service/apis"
+	"github.com/ethereum-optimism/optimism/op-service/txintent/bindings"
+	"github.com/ethereum-optimism/optimism/op-service/txintent/contractio"
+	"github.com/ethereum/go-ethereum/consensus/misc/eip1559"
 )
 
-// TestConfigurableMinBaseFee verifies that the configurable minimum base fee feature works correctly
-func TestConfigurableMinBaseFee(t *testing.T) {
-	t.Run("MinBaseFeeOff", func(t *testing.T) {
-		// when minBaseFee is 0, base fees can drop normally
-		runMinBaseFeeTest(t, configurableMinBaseFeeOffTestScenario)
+// TestConfigurableMinBaseFee verifies configurable minimum base fee using devstack presets (sysgo).
+func TestConfigurableMinBaseFee(gt *testing.T) {
+	t := devtest.SerialT(gt)
+	sys := presets.NewMinimal(t)
```

Comment:
just for my understanding, was the sysgo setup the missing piece to making this test work for in-memory runner? 

are there any other pieces i might be missing in my understanding?